Makefile

```diff
@@ -4,7 +4,8 @@ test:
 	set -e; for dir in $(ALL_GO_MOD_DIRS); do \
 	  echo "go test in $${dir}"; \
 	  (cd "$${dir}" && \
-	    go test ./... && \
+	    go test && \
+	    env GOOS=linux GOARCH=386 go test && \
```

Comment:
it's specifically 32bit ARM and 32bit MIPS where those previous issues fail, so depending on host platform this test may not actually trigger the previous issue

Reply:
Got it, but it worked fine in this case.